deeppavlov/dataset_readers/conll2003_reader.py

```diff
@@ -13,9 +13,11 @@ def read(self,
              data_path: str,
              dataset_name: str = None,
              provide_pos: bool = False,
+             iob: bool = False,
```

Comment:
If you're adding a new positional argument, you should add it to the end to not break existing calls